config/sniffer.js

```diff
@@ -16,5 +16,5 @@ module.exports = {
         }
     },
 
-    enable: []
+    enable: ['mandatory-object-comma']
```

Comment:
Why do you turn on only one rule?

Reply:
It is my bad. By default array is empty. I removed name of rule